baseplate/context/memcache/lib.py

```diff
@@ -0,0 +1,111 @@
+import logging
+import zlib
+
+from ..._compat import (
+    long,
+    pickle,
+    string_types,
+)
+
+
+"""Memcache serialization/deserialization (and compression) helper methods.
+
+Memcached can only store strings, so to store arbitrary objects we need to
+serialize them to strings and be able to deserialize them back to their
+original form. General purpose serialization and deserialization can be
+achieved with pickle_and_compress() and decompress_and_unpickle().
+
+"""
+
+
+FLAG_PICKLE = 1 << 0
```

Comment:
Hmm, if I make it an `Enum`, then I'd need to do like `flags |= Flags.ZLIB.value` to get the actual integer value. What do you think about doing:

```
class Flags(object):
    """Memcached client flags

    Flags are an arbitrary 16-bit unsigned integer that the memcache server
    stores along with the data and sends back when the item is retrieved.
    Clients may use this as a bit field to store data-specific information;
    this field is opaque to the server.

    """

    PICKLE = 1 << 0
    INTEGER = 1 << 1
    LONG = 1 << 2
    ZLIB = 1 << 3
```